airflow/providers/amazon/aws/transfers/s3_to_dynamodb.py

```diff
@@ -0,0 +1,313 @@
+
+    IndexName: str
+    KeySchema: KeySchema
+    Projection: Projection
+    ProvisionedThroughput: ProvisionedThroughput
+    OnDemandThroughput: OnDemandThroughput
+
+
+class S3ToDynamoDBOperator(BaseOperator):
+    """Load Data from S3 into a DynamoDB.
+
+    Data stored in S3 can be uploaded to a new or existing DynamoDB. Supported file formats CSV, DynamoDB JSON and
+    Amazon ION.
+
+
+    :param s3_bucket: The S3 bucket that is imported
+    :param s3_key: key prefix that imports single or multiple objects from S3
+    :param dynamodb_table_name: Name of the table that shall be created
+    :param dynamodb_key_schema: Primary key and sort key. Each element represents one primary key
+        attribute. AttributeName is the name of the attribute. KeyType is the role for the attribute. Valid values
+        HASH or RANGE
+    :param dynamodb_attributes: Name of the attributes of a table. AttributeName is the name for the attribute
+        AttributeType is the data type for the attribute. Valid values for AttributeType are
+        S - attribute is of type String
+        N - attribute is of type Number
+        B - attribute is of type Binary
+    :param dynamodb_tmp_table_prefix: Prefix for the temporary DynamoDB table
+    :param delete_dynamodb_tmp_table: If set, deletes the temporary DynamoDB table that is used for staging
+    :param use_existing_table: Whether to import to an existing non new DynamoDB table. If set to
+        true data is loaded first into a temporary DynamoDB table, then retrieved as chunks into memory and loaded
+        into the target table
+    :param client_token: ClientToken can be used for idempotent calls that have the same effect as one call.
+        A token is valid for 8 hours after the first request. Afterwards nay new request with the same token is
+        considered as a new request
+    :param input_format: The format for the imported data. Valid values for InputFormat are CSV, DYNAMODB_JSON
+        or ION
+    :param input_format_options: Extra options on how the input is formatted such as CSV delimiter options
+    :param input_compression: Compression options. Valid values for InputCompressionType are NONE, GZIP,
+        or ZSTD
+    :param billing_mode: Billing mode for the table. Valid values are PROVISIONED or PAY_PER_REQUEST
+    :param provisioned_throughput: Extra options for provisioned throughput settings
+    :param on_demand_throughput: Extra options for maximum number of read and write units
+    :param sse_specification: Extra options for server-side encryption
+    :param global_secondary_indexes: Extra options for server-side encryption
+    :param wait_for_completion: Whether to wait for cluster to stop
+    :param check_interval: Time in seconds to wait between status checks
+    :param max_attempts: Maximum number of attempts to check for job completion
+    :param aws_conn_id: The reference to the AWS connection details
+    """
+
+    template_fields: Sequence[str] = (
```

Comment:
I guess we can add other parameters as well

Reply:
Changed.